packages/entity-list/src/main.js

```diff
@@ -116,6 +116,18 @@ EntityListApp.propTypes = {
   ),
   disableSimpleSearch: PropTypes.bool,
   simpleSearchFields: PropTypes.string,
+  /**
```

Comment:
to be consistent i would omit these comments since its a duplication of informations from the readme